src/mmg2d/mmg2d6.c

```diff
@@ -309,8 +309,8 @@ int MMG2D_mmg2d6(MMG5_pMesh mesh, MMG5_pSol sol,MMG5_pSol met) {
   if ( mesh->info.iso ) {
     /* Check that the resulting mesh is manifold */
     if ( !MMG5_chkmanimesh(mesh) ) {
-      fprintf(stderr,"\n  ## No manifold resulting situation. Exit program.\n");
-      return 0;
+      fprintf(stdout,"\n  ## WARNING : No manifold resulting situation.\n");
+      return 1;
```

Comment:
We want to keep an error here: we would like to avoid loosing time to remesh a situation that the user probably wants to avoid.